features/step_definitions/upload_evaluation_steps.rb

```diff
@@ -0,0 +1,28 @@
+Given(/^User is on the import page$/) do
+  visit '/evaluation/import'
+end
+
+Then(/^User should be on the import page$/) do
+  current_path.should == "/evaluation/import"
+end
+
+When(/^User selects excel file$/) do
+  page.attach_file("data_file", Rails.root + 'spec/fixtures/StatisticsReport.xlsx')
+end
+
+When(/^User selects a non-excel file$/) do
+  page.attach_file("data_file", Rails.root + 'spec/fixtures/ruby-capybara.zip')
+end  
+
+Then(/^User should be on the evaluation page$/) do
+  current_path.should == "/evaluation"
```

Comment:
varieties do, such as when instead of then, or /evaluation vs /evaluation/import